velox/docs/functions/spark/map.rst

```diff
@@ -6,18 +6,18 @@ Map Functions
 
     Returns value for given ``key``, or ``NULL`` if the key is not contained in the map.
 
-.. spark:function:: map() -> map(unknown, unknown)
+.. spark:function:: map(K, V, K, V, ...) -> map(K,V)
 
-    Returns an empty map. ::
+    Returns a map created using the given key/value pairs. ::
 
-        SELECT map(); -- {}
+        SELECT map(1, 2, 3, 4); -- {1 -> 2, 3 -> 4}
 
-.. spark:function:: map(array(K), array(V)) -> map(K,V)
+.. spark:function:: map() -> map(unknown, unknown)
```

Comment:
Is unkown type included in the return map, or just empty map is returned? Maybe we can use a more accurate signature.

Reply:
I checked this is not valid signature, and for spark `map()` is converted to `{}` by spark optimizer, so this should be ok.

Reply:
![image](https://github.com/facebookincubator/velox/assets/10524738/06f5100a-2b1a-4871-baee-105819a897fb)